src/pages/Campaign/CampaignButtonWithOptions.tsx

```diff
@@ -105,7 +105,8 @@ export default function CampaignButtonWithOptions({
       disabled={disabled}
       ref={containerRef}
     >
-      <Trans>{type === 'enter_now' ? 'Enter now' : 'Claim Rewards'}</Trans>
+      {type === 'enter_now' && <Trans>Enter now</Trans>}
+      {type === 'claim_rewards' && <Trans>Claim Rewards</Trans>}
```

Comment:
We can write like this: `{type === 'enter_now' ? t`Enter now` : t`Claim Rewards`}`

Reply:
done